include/onnxruntime/core/providers/dml/dml_provider_factory.h

```diff
@@ -99,6 +99,18 @@ struct OrtDmlApi {
    * This API gets the D3D12 resource when an OrtValue has been allocated by the DML EP.
    */
   ORT_API2_STATUS(GetD3D12ResourceFromAllocation, _In_ OrtAllocator* provider, _In_ void* dml_resource, _Out_ ID3D12Resource** d3d_resource);
```

Comment:
Investigate winrt API breaking changes

Reply:
This doesn't break the winrt API since the BFC is disabled for custom ops, and without custom ops callers can only get resources for the resources that they created themselves.